### PR TITLE
CLOUDP-161495: disable bodyclose rule

### PR DESCRIPTION
**Pull request by wtrocki**
See: https://jira.mongodb.org/browse/CLOUDP-161495

TL:DR - there are so many false positives for bodyclose that this rule only works when stream is opened only within the same golang scope (same code block). Any functions, or external library response objects are reported as errors.
Considering the very poor maintenance of the rule (those issues were reported years ago) I recomend to disable rule rather than adding +400 exceptions. 

This PR is an alternative to current approach:
Supplying extra defer for all sdk calls (which would result in noop as body is closed already)

### CODE REVIEW

**Review by matt-condon — APPROVED** [review]
SGTM

**Comment by wtrocki** [comment]
@andreaangiolillo @fmenezes Looking for another opinion before we merge that change.

**Review by fmenezes — APPROVED** [review]
LGTM

**Pull request merged**
